src/Features/Core/Portable/CodeFixes/AddImport/AbstractAddImportCodeFixProvider.cs

```diff
@@ -265,6 +267,42 @@ internal abstract partial class AbstractAddImportCodeFixProvider<TSimpleNameSynt
         }
 
         /// <summary>
+        /// We ignore references that are in a directory that contains the names "Packages".
+        /// These directories are most likely the ones produced by NuGet, and we don't want
+        /// to offer to add .dll reference manually for dlls that are part of NuGet packages.
+        /// </summary>
+        private bool IsInPackagesDirectory(PortableExecutableReference reference)
+        {
+            const string Packages = nameof(Packages);
```

Comment:
Because it's just redundant for me to have the variable name and the string literal.   With a string literal there's also a chance i mispell things.  i.e. ```const string packages = "pakages";```.  This pattern avoids that because the compiler adds some checking to make sure the name on the right is matches the name i wrote on the left.

I don't expect these names to ever be renamed.